packages/protocol/migrations_sol/Migration.s.sol

```diff
@@ -11,6 +11,9 @@ import "@celo-contracts/common/interfaces/IProxy.sol";
 import "@celo-contracts/common/interfaces/IRegistry.sol";
 import "@celo-contracts/common/interfaces/IFreezer.sol";
 import "@celo-contracts/common/interfaces/IFeeCurrencyWhitelist.sol";
+// TODO(Arthur): import FeeCurrencyDirectory interface?
```

Comment:
Do we need this todo ?

Reply:
No, I removed various `TODO` comments I left for myself in this commit https://github.com/celo-org/celo-monorepo/pull/10992/commits/18eb1638618457022c443c6571968eb47dbe9848